org.scala-ide.play2/src/org/scalaide/play2/templateeditor/processing/TemplateProcessingProvider.scala

```diff
@@ -0,0 +1,24 @@
+package org.scalaide.play2.templateeditor.processing
+
+import org.eclipse.core.runtime.RegistryFactory
+import org.scalaide.play2.PlayPlugin
+import org.scalaide.play2.properties.PlayPreferences
+
+object TemplateProcessingProvider {
+  val ExtensionPointId = "org.scalaide.play2.template.processing"
+
+  def templateProcessing(): TemplateProcessing = {
```

Comment:
Actually, I noticed that you only load one extension here anyway and since we are probably always the provider of this extension, there is not really a need to make this more safe.